site/guide/model-documentation/work-with-content-blocks.qmd

```diff
@@ -3,27 +3,57 @@ title: "Work with content blocks"
 date: last-modified
 ---
 
-Make edits to your model documentation or validation reports by adding or removing content blocks directly in the online editor. 
+Make edits to your model documentation, validation reports, or ongoing monitoring plans by adding or removing content blocks directly in the online editor. 
 
 ## What are content blocks? 
 
-Content blocks provide you with sections that are part of a template. You can think of these sections as an empty canvas that you fill in with text and test results. Multiple sections are joined to create a longer document with a table of contents that has different heading and subheading levels, such as "1.," "1.1.," and so on. 
+Content blocks provide you with sections that are part of a template. 
 
-Types of content blocks: 
+- You can think of these sections as an empty canvas that you fill in with text and test results. 
+- Multiple sections are joined to create a longer document with a table of contents that has different heading and subheading levels, such as "1.," "1.1.," and so on. 
 
-Simple text block
-: Can be added anywhere on model documentation or validation reports and edited to include additional documentation in text format.
+#### Content block types
+
+simple text block
+: Can be added anywhere on model documentation, validation reports, or ongoing monitoring plans and edited to include additional documentation in text format.
+
+test-driven block
+: Can be within model documentation and ongoing monitoring plans to display one of the supported test results collected by the {{< var validmind.developer >}}.
+
+metric over time block
+: Can be added within model documentation and ongoing monitoring plans to display one of the supported metric over time results collected by the {{< var vm.developer >}}.
+
+### Content editing toolbar
+
+![Content editing toolbar](/guide/model-documentation/content-editing-toolbar.png){fig-alt="A screenshot showing the content editing toolbar"}
+
+The content editing toolbar is a rich text editor that enables you to:
+
+- Undo or redo changes
+- Format your text, including adding hyperlinks and code blocks
+- Insert LaTex formulas[^1]
+- Attach images via upload or URL
+
+You can also use the toolbar to suggest changes, save named versions of content, and leave comments.[^2] 
+
+::: {.content-hidden when-format="html"}
+
+<!-- As per Nik on PR#233 (https://github.com/validmind/documentation/pull/233) we are hiding this content until we are sure we want to expose the documentation surrounding the Story to users.  When it's ready to be revealed, simply remove this comment and the `.content-hidden` block it's wrapped in. -->
+
+::: {.callout title="Stuck on where to start?"}
+Use ValidMind to assist you with generating content via AI!^[[Generate with AI (BETA)](#generate-with-ai-beta)]
```

Comment:
I think this feature is still behind a feature flag — it might be good to ask Kristof or Mehdi if we should start including this in our docs. I know the screen capture shows the AI button and perhaps it doesn't matter at this point, but I have been asked to remove references to this feature before.

Reply:
Cool, I will just tag him in a DM.